File: 백준/Bronze/9086. 문자열/문자열.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int t = Integer.parseInt(br.readLine());
        String result = "";
        for (int i = 0; i < t; i++) {
            String[] arr = br.readLine().split("");
            result += arr[0] + arr[arr.length -1] + "\n";
        }
        System.out.println(result);

    }
}

File: 백준/Bronze/9086. 문자열/문자열.kt
import java.io.BufferedReader
import java.io.InputStreamReader

// kotlinc -include-runtime -d output.jar main.kt
// java -jar output.jar

fun main() = with(BufferedReader(InputStreamReader(System.`in`))) {
    val t = readLine().toInt()
    var result = ""
    for(i in 1..t) {
        val arr = readLine().trim().toCharArray()
        val first = arr[0]
        val last = arr[arr.size -1]
        result += "$first$last\n"
    }
    println(result)
}

